.github/workflows/releaser.yaml

```
@@ -8,6 +8,16 @@ on:
         required: true
         default: 'main'
         type: string
+      PROJECT_NAME:
+        description: 'The name of the project to release.'
+        required: true
+        default: 'signoz'
```

actionlint (1.7.4)

input "PROJECT_NAME" of workflow_call event has the default value "signoz", but it is also required. if an input is marked as required, its default value will never be used

(events)